motionplan/tpspace/ptg.go

```diff
@@ -95,7 +95,14 @@ func ComputePTG(simPTG PTG, alpha, dist, diffT float64) ([]*TrajNode, error) {
 		alphaTraj[len(alphaTraj)-1].AngVelRPS = w
 
 		alphaTraj = append(alphaTraj, nextNode)
-		distTravelled += math.Copysign(math.Max(diffT, math.Abs(v)*diffT), dist)
+		displacement := diffT
+		switch {
+		case v != 0:
+			displacement = math.Copysign(math.Abs(v)*diffT, dist)
+		case w != 0:
```

Comment:
Do we not need to do something different for the rotational distance?  Also can there ever be a case when both cases are true?  If so wouldnt we want to be adding both?

Reply:
That shouldn't matter for displacement. If you're driving 0.5 m/s, then you should displace 500mm in a second regardless of of whether you are going 500mm straight ahead, or driving halfway around a circle with a 1 meter circumference.